crates/rspack_plugin_javascript/src/visitors/dependency/parser/walk.rs

```diff
@@ -350,6 +350,7 @@ impl<'parser> JavascriptParser<'parser> {
   }
 
   fn walk_variable_declaration(&mut self, decl: &VarDecl) {
+    self.enter_assign = true;
```

Comment:
Should this in the for loop? since `= false` is in the for loop

Reply:
The logic within `﻿ApiScanner` was simply replicated, and additionally, the ﻿`walk_variable_declaration` will not influence the for-loop body.